guides/common/modules/con_additional-maintenance-required-for-load-balancing-in-project.adoc

```diff
@@ -0,0 +1,10 @@
+[id="additional-maintenance-required-for-load-balancing-in-project_{context}"]
+= Additional maintenance required for load balancing in {Project}
+
+Configuring {SmartProxies} to use a load balancer results in a more complex environment and requires additional maintenance.
+
+The following additional steps are required for load balancing:
+
+* You must ensure that all {SmartProxies} have the same content views and synchronize all {SmartProxies} to the same content view versions
+* You must upgrade each {SmartProxy} in sequence
+* You must backup each {SmartProxy} that you configure regularly
```

Comment:
```suggestion
* You must back up each {SmartProxy} that you configure regularly.
```

The noun is "backup". The verb is "back up". It's like "login" (login credentials) vs "log in" (log in to the server).